app/views/support_interface/sessions/confirm_environment.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :title, 'Confirm environment' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body-l">To make sure you intend to make changes to <strong><%= HostingEnvironment.environment_name %></strong>, type <%= HostingEnvironment.environment_name %> below:</p>
+
+    <%= form_with(url: support_interface_confirm_environment_path) do |f| %>
+      <%= f.hidden_field :from, value: params[:from] %>
+      <%= f.govuk_text_field :environment, label: { text: 'Type it', size: 'm' } %>
```

Comment:
Could probably combine the guidance with the label text, maybe something like:

## Confirm action

This is the **production** environment.

Type `production` to confirm that you want to proceed
[_________________________]

Reply:
(Clearly I’m not a content designer 😂)